docs/DeveloperGuide.adoc

```diff
@@ -80,11 +79,10 @@ image::LogicClassDiagram.png[]
 [discrete]
 ==== How the architecture components interact with each other
 
-This section explains the interactions amongst the different architecture components using an example.
-The _Sequence Diagram_ below shows how the components interact with each other
-for the scenario where the user issues the command `delete 1`.
+The _Sequence Diagram_ below shows how the components interact with each other for the scenario when a user
+issues the command `edit ic/S5678908D n/Mr Lim`.
```

Comment:
Btw the current format is `edit S45923191A` without the prefix `ic` can help amend?